docker-compose.yml

```diff
@@ -56,79 +61,77 @@ services:
   registry:
     build:
       context: registry
-      args: 
-        REGISTRY_USERNAME: username
-        REGISTRY_PASSWORD: password
-    environment: 
+      args:
+        REGISTRY_USERNAME: $TINKERBELL_REGISTRY_USER
+        REGISTRY_PASSWORD: $TINKERBELL_REGISTRY_PASS
+    environment:
       REGISTRY_HTTP_ADDR: 0.0.0.0:443
       REGISTRY_HTTP_TLS_CERTIFICATE: /certs/server.pem
       REGISTRY_HTTP_TLS_KEY: /certs/server-key.pem
       REGISTRY_AUTH: htpasswd
       REGISTRY_AUTH_HTPASSWD_REALM: "Registry Realm"
       REGISTRY_AUTH_HTPASSWD_PATH: /auth/htpasswd
-    volumes: 
+    volumes:
       - ./certs:/certs
+    depends_on:
+      - fluentbit
     logging:
       driver: fluentd
       options:
         tag: registry
     network_mode: host
 
   boots:
-    build:
-      context: ../boots
+    image: quay.io/packet/boots:workflow
     network_mode: host
-    command: -dhcp-addr 0.0.0.0:67 -tftp-addr 127.0.0.1:69 -http-addr 127.0.0.1:80 -log-level DEBUG
+    command: -dhcp-addr 0.0.0.0:67 -tftp-addr $HOST_IP:69 -http-addr $HOST_IP:80 -log-level DEBUG
     environment:
-      API_AUTH_TOKEN: ${PACKET_API_AUTH_TOKEN:-PcyR6MvHb7wMmyYf9p8dJ2Dvnb9HxX8E}
-      API_CONSUMER_TOKEN: ${PACKET_CONSUMER_TOKEN:-djR2TAvbnkY92i8Ea2KFMZW6MusW1fk7qzeCUHgtnQRSsXnqxoCr6V2vhSxpqASf}
+      API_AUTH_TOKEN: ${PACKET_API_AUTH_TOKEN:-########################}
+      API_CONSUMER_TOKEN: ${PACKET_CONSUMER_TOKEN:-####################}
       FACILITY_CODE: ${FACILITY:-lab1}
       PACKET_ENV: ${PACKET_ENV:-testing}
       PACKET_VERSION: ${PACKET_VERSION:-5efab5ef3a42cb88f2d54f4ed3201c2dd6797b7d}
-      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-9b78d0ad01d1467aa92c49c3a349b79d}
-      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-0}
-      MIRROR_HOST: ${MIRROR_HOST:-127.0.0.1}
-      DNS_SERVERS: 8.8.8.8
-      PUBLIC_IP: 127.0.0.1
-      BOOTP_BIND: 127.0.0.1:67
-      HTTP_BIND: 127.0.0.1:80
-      SYSLOG_BIND: 127.0.0.1:514
-      TFTP_BIND: 127.0.0.1:69
-      DOCKER_REGISTRY: 127.0.0.1
-      REGISTRY_USERNAME: username
-      REGISTRY_PASSWORD: password
-      TINKERBELL_GRPC_AUTHORITY: 127.0.0.1:42113
-      TINKERBELL_CERT_URL: http://127.0.0.1:42114/cert
-      ELASTIC_SEARCH_URL: 127.0.0.1:9200
+      ROLLBAR_TOKEN: ${ROLLBAR_TOKEN:-#################################}
+      ROLLBAR_DISABLE: ${ROLLBAR_DISABLE:-1}
+      MIRROR_HOST: ${NGINX_IP:-127.0.0.1}
+      CACHER_GRPC_AUTHORITY: 127.0.0.1:42111
+      CACHER_CERT_URL: http://127.0.0.1:42112/cert
+      DNS_SERVERS: "147.75.207.207,147.75.207.208"
+      PUBLIC_IP: $HOST_IP
+      BOOTP_BIND: $HOST_IP:67
+      HTTP_BIND: $HOST_IP:80
+      SYSLOG_BIND: $HOST_IP:514
+      TFTP_BIND: $HOST_IP:69
+      DOCKER_REGISTRY: $HOST_IP
+      REGISTRY_USERNAME: $TINKERBELL_REGISTRY_USER
+      REGISTRY_PASSWORD: $TINKERBELL_REGISTRY_PASS
+      TINKERBELL_GRPC_AUTHORITY: $HOST_IP:42113
+      TINKERBELL_CERT_URL: http://$HOST_IP:42114/cert
+      ELASTIC_SEARCH_URL: $HOST_IP:9200
     depends_on:
       - tinkerbell
```

Comment:
I don't think if `boots` should depend on `tinkerbell`. It may be applicable vice-versa though.

Reply:
It was already there I just changed the name from server to tinkerbell.